vpdq/python/tests/test_vpdq_hash.py

```diff
@@ -9,7 +9,7 @@
 SAMPLE_HASH_FOLDER = Path("vpdq/sample-hashes")
 SAMPLE_VIDEOS = Path("tmk/sample-videos")
 DISTANCE_TOLERANCE = 10
-QUALITY_TOLERANCE = 50
```

Comment:
blocking: Why are we updating this to 80 again?

Reply:
"Adjust test parameters with CPP github workflow." CPP reg test uses 80. 50 may fail on various ffmpeg versions caused by that low-quality video.

Reply:
Having a test QUALITY_TOLERANCE higher than the used in prod `VPDQ_QUALITY_THRESHOLD = 50` doesn't make sense to me. (Same goes for what we do in C++ land)
I would prefer we skip the low-quality video for a regression test as it is more import the test reflects the production state of the library.

Reply:
The DISTANCE_TOLERANCE is lower than the used in prod. I can adjust those to be the same as the prod data. Is it better?

Reply:
Sure, for the reg test, let's match prod defaults.
Was it a deliberate decision not to do so originally?

Reply:
Yes, we had discussion on this for smaller DISTANCE_TOLERANCE for validation process.

Reply:
Ah I see, I view implementation validation as different from regression testing:
validation: two implementation of the vpdq algorithm ideally output nearly identical hashes (good to have much for strict params)
regression: adjustments, refactors, etc to code do not result in large changes in production behavior (can be more strict or just match production params)